File: 백준/Bronze/30676. 이 별은 무슨 색일까/이 별은 무슨 색일까.cc
#include <iostream>
using namespace std;
int main(void) {
    int n;
    scanf("%d", &n);
    string r;
    if (n >= 620)
        r = "Red";
    else if (n >= 590)
        r = "Orange";
    else if (n >= 570)
        r = "Yellow";
    else if (n >= 495)
        r = "Green";
    else if (n >= 450)
        r = "Blue";
    else if (n >= 425)
        r = "Indigo";
    else if (n >= 380)
        r = "Violet";
    cout << r;
    return 0;
}

File: 백준/Bronze/30676. 이 별은 무슨 색일까/이 별은 무슨 색일까.py
import sys
input=lambda:sys.stdin.readline().rstrip()
n=int(input())
r=""
if n>=620:
    r="Red"
elif n>=590:
    r="Orange"
elif n>=570:
    r="Yellow"
elif n>=495:
    r="Green"
elif n>=450:
    r="Blue"
elif n>=425:
    r="Indigo"
elif n>=380:
    r="Violet"
print(r)